IPython/utils/tests/test_text.py

```diff
@@ -180,6 +180,7 @@ def test_LSString():
     lss = text.LSString("abc\ndef")
     nt.assert_equal(lss.l, ['abc', 'def'])
     nt.assert_equal(lss.s, 'abc def')
+    nt.assert_in("path", sys.modules)
```

Comment:
Okay, I think I understand better. But to be sure to have some value back from `lss.get_paths()` I will need to change lss as neither `abc` or `def` are correct path values. I'm thinking about something like : 
```python
lss = text.LSSstring("\n".join(os.getcwd(), os.getcwd())
nt.assert_equal(lss.p, [os.getcwd(), os.getcwd()])
```
Using `os.getcwd()` ensures we have a proper path in any case.